tests/integration/integration_steps/fabric_steps.py

```diff
@@ -105,3 +111,27 @@ def assert_data_model_in_fabric():
 def assert_data_model_update():
     # Assert the data model changes including versions and last updated timestamps are propagated to a Fabric lakehouse
     pass
+
+
+def assert_events_data_in_fabric(
+    events_path: str,
+    events_dataframe: pd.DataFrame,
+    azure_credential: DefaultAzureCredential,
+):
+    # Assert events data is populated in a Fabric lakehouse
+    events_from_lakehouse = read_deltalake_timeseries(events_path, azure_credential)
+
+    # Prepare the lakehouse data for comparison
+    events_from_lakehouse = (
+        events_from_lakehouse.drop(columns=EVENT_CDF_COLUMNS)
+        .sort_values(by=["startTime"])
```

Comment:
Shall "startTime" be set as a constant?

Reply:
Yes.  At some point we should refactor the fabric assertions to be reusable for the different data types and be able to pass in the columns we want to drop and the ones we want to sort by

Reply:
Done